src/main/java/com/google/devtools/build/lib/runtime/commands/RunCommand.java

```diff
@@ -511,9 +511,12 @@ public BlazeCommandResult exec(CommandEnvironment env, OptionsParsingResult opti
             ByteString.copyFrom(workingDir.getPathString(), StandardCharsets.ISO_8859_1));
 
     if (OS.getCurrent() == OS.WINDOWS) {
+      boolean isBinary = true; // The first element in `cmdLine` is the binary, which must not be escaped.
```

Comment:
I'd put the comment right before the call to windowsEscapeArg, and also split up the ternary operator into an actual if/else statement.

Reply:
Done + altered the comment to reflect its new location.